e2e/app/playwright.config.js

```diff
@@ -0,0 +1,19 @@
+import { defineConfig, devices } from '@playwright/test';
+
+import baseConfig from '../playwright.config';
+
+export default defineConfig({
+  ...baseConfig,
+  use: {
+    ...baseConfig.use,
+    baseUrl: baseConfig.use.baseUrl || "localhost:3000"
+  },
+  projects: [
+    ...baseConfig.projects,
+    // add Safari for derived app config
+    {
+      name: "webkit",
+      use: { ...devices["Desktop Safari"] },
+    },
```

Comment:
do we need safari? i thought we're just gonna start with the two browsers defined in the default config

Reply:
Just an example of overriding a browser - if we think it's obvious how to do that, let's remove.